test/spec/widget.datetime.spec.js

```diff
@@ -46,7 +46,7 @@ describe( 'datetimepicker widget', () => {
             fakeDateInput.dispatchEvent( new Event( 'change' ) );
 
             // timezone info will be added by engine
-            expect( input.value ).to.equal( '2012-01-01T01:01:00.000' );
+            expect( input.value.endsWith( '-07:00' ) ).to.equal( false );
```

Comment:
Seems like `expect( input.value.startsWith( '2012-01-01T01:01' ) ).to.equal( true );` would be clear?